ci-operator/step-registry/baremetalds/packet/setup/baremetalds-packet-setup-commands.sh

```diff
@@ -111,6 +111,21 @@ cat > packet-setup.yaml <<-EOF
             ssh "\${SSHOPTS[@]}" "root@\${IP}" hostname && break
             sleep 10
         done
+
+        ssh "\${SSHOPTS[@]}" "root@\${IP}" bash - << "EOF"
+          source /etc/os-release
+
+          if [[ "\${ID}" == "centos" ]] && [[ "\${NAME}" != "CentOS Stream" ]]; then
```

Comment:
Maybe we can remove this if condition to test the intention below is working. In any case, it is hardcoded `centos`.

Reply:
sounds like a good idea
removing it for now